parts/k8s/kubernetesconfigs.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
+MASTER_INDEX=$(hostname | tail -c 2)
+MASTER_VM_NAME=$(hostname)
```

Comment:
This script is common to both agents and masters so for clarity you could either a) define the variables inside the function that calls them (the etcd configuration only happens on master nodes) b) only define these vars on master nodes, or c) rename them to make them more general since they apply to both: NODE_INDEX and NODE_NAME or something like that